trestle/oscal/common.py

```diff
@@ -1282,64 +1551,154 @@ class Protocol(OscalBaseModel):
+    ) = Field(
         ...,
-        description="A textual label that uniquely identifies the part's semantic type, which exists in a value space qualified by the ns.",
-        title='Part Name',
+        description=
+        'A machine-oriented, globally unique identifier with cross-instance scope that can be used to reference this component elsewhere in this or other OSCAL instances. The locally defined UUID of the component can be used to reference the data item locally or globally (e.g., in an imported OSCAL instance). This UUID should be assigned per-subject, which means it should be consistently used to identify the same subject across revisions of the document.',
+        title='Component Identifier',
     )
-    ns: Optional[AnyUrl] = Field(
-        None,
-        description="An optional namespace qualifying the part's name. This allows different organizations to associate distinct semantics with the same name.",
-        title='Part Namespace',
+    type: Union[constr(regex=r'^\S(.*\S)?$'), SystemComponentTypeValidValues] = Field(
+        ..., description='A category describing the purpose of the component.', title='Component Type'
     )
-    class_: Optional[constr(regex=r'^[_A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD][_A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD\-\.0-9\u00B7\u0300-\u036F\u203F-\u2040]*$')] = Field(
-        None,
-        alias='class',
-        description="An optional textual providing a sub-type or characterization of the part's name, or a category to which the part belongs.",
-        title='Part Class',
+    title: str = Field(..., description='A human readable name for the system component.', title='Component Title')
+    description: str = Field(
+        ...,
+        description='A description of the component, including information about its function.',
+        title='Component Description'
     )
-    title: Optional[str] = Field(
-        None,
-        description='An optional name given to the part, which may be used by a tool for display and navigation.',
-        title='Part Title',
+    purpose: Optional[str] = Field(
+        None, description='A summary of the technological or business purpose of the component.', title='Purpose'
     )
     props: Optional[List[Property]] = Field(None)
-    prose: Optional[str] = Field(
-        None,
-        description='Permits multiple paragraphs, lists, tables etc.',
-        title='Part Text',
+    links: Optional[List[Link]] = Field(None)
+    status: Status = Field(..., description='Describes the operational status of the system component.', title='Status')
+    responsible_roles: Optional[List[ResponsibleRole]] = Field(None, alias='responsible-roles')
+    protocols: Optional[List[Protocol]] = Field(None)
+    remarks: Optional[str] = None
```

Comment:
The decision was made by the original trestle developers to **not** employ classes that comprised only` __root__:` because of little benefit derived. There are a whole bunch besides Remarks. Also a conscious decision was made to **not** remove these unused classes strictly as a documentary device, I believe.